matchbox_demo/src/main.rs

```diff
@@ -21,6 +21,9 @@ enum AppState {
 
 const SKY_COLOR: Color = Color::rgb(0.69, 0.69, 0.69);
 
+#[derive(Default, Resource)]
+struct SocketResource(Option<WebRtcSocket>);
```

Comment:
**question** Is there a better way of doing this, perhaps a `bevy` feature for this crate that applies this macro if enabled?

Reply:
Not really, it has to be an `Option` so we can take ownership of it. In the lobby system.

Could perhaps rethink it a little bit, but this is fine just to get the port done.